### PR TITLE
Openapi

### PR DESCRIPTION
**Pull request by tokuhirom**
- [x] set deprecated tag for existing methods

### CODE REVIEW

**Comment by tokuhirom** [comment]
- examples/access_token_helper/main.go was removed due to hard to maintain.

File: examples/echo_bot/server.go [hunk 1]
```diff
@@ -21,48 +21,89 @@ import (
...
+			switch e := event.(type) {
+			case webhook.MessageEvent:
+				switch message := e.Message.(type) {
+				case webhook.TextMessageContent:
+					if _, err = bot.ReplyMessage(
+						&messaging_api.ReplyMessageRequest{
+							ReplyToken: e.ReplyToken,
+							Messages: []messaging_api.MessageInterface{
+								messaging_api.TextMessage{
+									Text: message.Text,
```
Comment (tokuhirom) [inline examples/echo_bot/server.go:65]: "type" filed is filled by MarshalJSON.

File: linebot/messaging_api/api_messaging_api.go [hunk 2]
```diff
@@ -0,0 +1,3040 @@
...
+	channelToken string
+	ctx          context.Context
+}
+
+// MessagingApiAPIOption type
+type MessagingApiAPIOption func(*MessagingApiAPI) error
+
+// New returns a new bot client instance.
+// Deprecated: Use OpenAPI based classes instead.
+func NewMessagingApiAPI(channelToken string, options ...MessagingApiAPIOption) (*MessagingApiAPI, error) {
```
Comment (Yang-33) [inline linebot/messaging_api/api_messaging_api.go:52]: Is this a mistake...? The new one doesn't seem to be deprecated...? (I guess generating code again resolve this)

File: linebot/version.go [hunk 3]
```diff
@@ -17,3 +17,8 @@
 package linebot
 
 const version = "7.21.0"
+
+// Deprecated: Use OpenAPI based classes instead.
```
Comment (Yang-33) [inline linebot/version.go:21]: probably it's not deprecated ...?

File: .github/workflows/go.yml [hunk 4]
```diff
@@ -31,5 +31,14 @@ jobs:
     - name: go vet
       run: go vet $(go list ./... | grep -v /examples/)
```
Comment (Yang-33) [inline .github/workflows/go.yml:33]: Let's generate code at first, then test changes!

File: .github/workflows/go.yml [hunk 5]
```diff
@@ -31,5 +31,14 @@ jobs:
     - name: go vet
```
Comment (Yang-33) [inline .github/workflows/go.yml:31]: Could you update README to explain how to use new sdk code ...?

**Review by Yang-33 — COMMENTED** [review]
LGTM with minor comments!!

File: examples/echo_bot/server.go (hunk 1 above)
Comment (Yang-33) [reply examples/echo_bot/server.go:65]: Very useful!

File: linebot/messaging_api/api_messaging_api.go (hunk 2 above)
Comment (tokuhirom) [reply linebot/messaging_api/api_messaging_api.go]: I regenerated code and it was gone :)

File: linebot/version.go (hunk 3 above)
Comment (tokuhirom) [reply linebot/version.go]: Yep.

File: .github/workflows/go.yml (hunk 4 above)
Comment (tokuhirom) [reply .github/workflows/go.yml:41]: Done

File: .github/workflows/go.yml (hunk 5 above)
Comment (tokuhirom) [reply .github/workflows/go.yml:45]: Done!

File: README.md [hunk 6]
```diff
@@ -34,38 +34,49 @@ import (
 )
 
 func main() {
-	bot, err := linebot.New("<channel secret>", "<channel access token>")
+	bot, err := messaging_api.NewMessagingApiAPI(
```
Comment (Yang-33) [inline README.md:37]: how about adding `import "github.com/line/line-bot-sdk-go/v7/linebot/messaging_api"`?

File: README.md [hunk 7]
```diff
@@ -34,38 +34,49 @@ import (
...
-## How to start ##
+## Getting Started ##
 
-The LINE Messaging API uses the JSON data format.
-```ParseRequest()``` will help you to parse the ```*http.Request``` content and return a slice of Pointer point to Event Object.
+The LINE Messaging API primarily utilizes the JSON data format. To parse the incoming HTTP requests, the `webhook.ParseRequest()` method is provided. This method reads the `*http.Request` content and returns a slice of pointers to Event Objects.
 
 ```go
-events, err := bot.ParseRequest(req)
+cb, err := webhook.ParseRequest(req)
```
Comment (Yang-33) [inline README.md:64]: how about adding `import "github.com/line/line-bot-sdk-go/v7/linebot/webhook"`?

**Review by Yang-33 — APPROVED** [review]
LGTM! Thank you for your hard work 🙇🙇🙇

File: README.md (hunk 6 above)
Comment (tokuhirom) [reply README.md:37]: good catch. i added import statement.

File: README.md (hunk 7 above)
Comment (tokuhirom) [reply README.md:68]: ditto

**Pull request merged**